frontend/packages/ux-editor/src/components/Properties/DataModelBindings.tsx

```diff
@@ -39,18 +38,14 @@ export const DataModelBindings = (): React.JSX.Element => {
     setMultipleAttachments(updatedValue);
     const updatedComponent = {
       ...formItem,
+      itemType: 'COMPONENT' as const,
       dataModelBindings: {
-        simpleBinding: !updatedValue ? '' : undefined,
-        list: updatedValue ? '' : undefined,
+        simpleBinding: { field: !updatedValue ? '' : undefined, dataType: '' },
+        list: updatedValue ? { field: '', dataType: '' } : undefined,
       },
-    };
-    handleUpdate(
-      updatedComponent as FormItem<ComponentType.FileUpload | ComponentType.FileUploadWithTag>,
-    );
-    debounceSave(
-      formItemId,
-      updatedComponent as FormItem<ComponentType.FileUpload | ComponentType.FileUploadWithTag>,
-    );
+    } as typeof formItem;
```

Comment:
```suggestion
    } as typeof FormComponent;
```
Should we explicit say it is a FormComponent here instead of formItem - which can be a FormContainer as well?